conda/recipes/rapids-xgboost/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  rapids_version: ${{ env.get("GIT_DESCRIBE_TAG", default="0.0.0.dev") | replace("v", "")}}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: rapids-xgboost
+  version: ${{ rapids_version }}
+
+source:
+  path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+
+requirements:
+  host:
+    - cuda-version =${{ cuda_version }}
+    - pip
+    - python =${{ py_version }}
+  run:
+    - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+    - nccl ${{ nccl_version }}
+    - python
+    - libxgboost ${{ xgboost_version }} rapidsai_h*
+    - xgboost ${{ xgboost_version }} rapidsai_py*
+    - if: cuda_major == "11"
+      then: cudatoolkit
+  ignore_run_exports:
+    by_name:
+      - python_abi
+      - if: cuda_major == "11"
+        then: cudatoolkit
```

Comment:
```suggestion
```

Now that CUDA 11 support is getting dropped on 25.08 (https://github.com/rapidsai/build-planning/issues/184), I think we could safely drop all the `if: cuda_major == "11"` stuff in any files you're already touching in this PR.

Notice that there aren't any CUDA 11 builds at https://github.com/rapidsai/integration/actions/runs/15421036533/job/43395979955?pr=772

<img width="1065" alt="image" src="https://github.com/user-attachments/assets/1117910b-2532-4697-8cbe-8b3225665d4a" />